frontend-react/src/components/MessageTracker/QualityFilters.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+
+import { Destination } from "../../resources/ActionDetailsResource";
+
+//TODO: do we want to create a new interface here or use the existing Destination?
+type QualityFiltersDisplayProps = {
+    qualityFilters: Destination[] | undefined;
+};
+
+export const QualityFilters = ({
+    qualityFilters,
+}: QualityFiltersDisplayProps) => {
+    return (
+        <>
+            <section className="margin-bottom-5">
+                <h3>Jurisdictions:</h3>
+                {qualityFilters?.map((d) => (
+                    <React.Fragment key={d.organization_id}>
```

Comment:
Today I learned you can use the `key` prop on a fragment. 😮

Reply:
Super clutch -- I didn't know `<React.Fragment>` would render the same as `<>` so I've always had to wrap my fragments in a component and key the component instead. Glad I saw this so I don't have to anymore!